docs/data/base/components/select/select.md

```diff
@@ -8,14 +8,14 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-
 
 # Unstyled Select
 
-<p class="description">The select components let you create lists of options for users to choose from.</p>
+<p class="description">The Select components let you create lists of options for users to choose from.</p>
 
 ## Introduction
 
 A select is a UI element that gives users a list of options to choose from.
 
-MUI Base offers two components to replace the native HTML `<select>` tag: `SelectUnstyled` and `MultiSelectUnstyled`.
-It also includes `OptionUnstyled` for creating the options on the list, and `OptionGroupUnstyled` for grouping those options.
+MUI Base offers two components to replace the native HTML `<select>` tag: Unstyled Select and Unstyled Multi-select.
```

Comment:
What would be the correct casing in compound words? We use Click-Away Listener but Multi-select.

Reply:
Great question @michaldudak. [Popular style guides](https://titlecaseconverter.com/blog/capitalizing-hyphenated-words/) leave their answers somewhat open-ended. [Google's documentation style guide](https://developers.google.com/style/capitalization#capitalization-and-hyphenated-words) recommends capitalizing every word in a compound proper noun—so we should probably go with "Multi-Select."

I was actually thinking about calling it "Multiselect" but there are 10x more search results for "Multi-Select," so I think that's preferable.